ci-operator/step-registry/telcov10n/functional/slcm/gather-report/telcov10n-functional-slcm-gather-report-commands.sh

```
@@ -3,3 +3,10 @@ set -e
 set -o pipefail
 
 echo "Gathering report..."
+export GDRIVE_FOLDER_NAME="${JOB_NAME}"
+export GDRIVE_PARENT_ID="$(cat /var/reporter/GDRIVE_FOLDER_ID)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)